repository/src/main/java/org/alfresco/service/cmr/repository/ScriptProcessor.java

```diff
@@ -73,7 +73,17 @@ public interface ScriptProcessor extends Processor
      * @return Obejct   the result of the script 
      */
     public Object executeString(String script, Map<String, Object> model);
-    
+
+    /**
+     * Execute script string
+     *
+     * @param script    the script string
+     * @param model     the context model
+     * @param secure    the flag that indicates if string is considered secure to be executed
```

Comment:
We use the variable name `secure` in multiple locations.  Can we make sure that what it means is adequately described in the javadoc? Someone coming to this new may not understand what we mean by "is considered secure to be executed"

Reply:
Description has been improved.